releases/client/3.1/settings

```diff
@@ -0,0 +1,3 @@
+FOREMAN_VERSION="3.1"
+FULLVERSION=$FOREMAN_VERSION
+OSES="rhel6 rhel7 el8 sles11 sles12"
```

Comment:
We normally always have line endings in files.

Reply:
Done, I was not aware of that.